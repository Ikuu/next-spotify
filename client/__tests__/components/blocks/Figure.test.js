import React from 'react';
import { create } from 'react-test-renderer';
import Figure from '../../../components/blocks/Figure';

test('Figure renders', () => {
  const tree = create(<Figure caption="Test"><img src="www.test.com" alt="alt" /></Figure>).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Figure doesnt renders', () => {
  const tree = create(<Figure caption="Test" />).toJSON();
  expect(tree).toMatchSnapshot();
});

