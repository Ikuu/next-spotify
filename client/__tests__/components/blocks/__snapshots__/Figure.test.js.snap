// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Figure doesnt renders 1`] = `null`;

exports[`Figure renders 1`] = `
<figure>
  <img
    alt="alt"
    src="www.test.com"
  />
  <figcaption>
    Test
  </figcaption>
</figure>
`;
